logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/type/UserTypeMapper.kt

```diff
@@ -54,6 +54,7 @@ class DomainUserTypeMapperImpl : DomainUserTypeMapper {
             UserTypeEntity.OWNER -> owner
             UserTypeEntity.ADMIN -> admin
             UserTypeEntity.SERVICE -> service
+            else -> none
```

Comment:
I'd rather be exhaustive here and do `UserTypeEntity.NONE -> none` instead as it might be that we add new user type entities in the future that will be silently swallowed here, right?